app/uk/gov/hmrc/merchandiseinbaggage/views/layouts/govukLayout.scala.html

```diff
@@ -30,7 +30,6 @@
         govukBackLink: GovukBackLink,
         languageSelection: components.languageSelection,
         hmrcReportTechnicalIssueHelper: HmrcReportTechnicalIssueHelper,
-        hmrcTrackingConsentSnippet: HmrcTrackingConsentSnippet,
```

Comment:
This was removed to avoid duplicate IDs as [HmrcHead](https://github.com/hmrc/play-frontend-hmrc/blob/main/play-frontend-hmrc-play-30/src/main/twirl/uk/gov/hmrc/hmrcfrontend/views/helpers/HmrcHead.scala.html#L24) has been added to Layout which includes it.